backend/tests/routes/broadcast/deleteBroadcast.test.js

```diff
@@ -0,0 +1,63 @@
+const chai = require('chai');
+const { expect } = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../index');
+
+chai.use(chaiHttp);
+
+// Test for delete Broadcast
+describe('Test for delete Broadcast:', () => {
+  let token = '';
+
+  // Step 1 - login as admin
+  it('log in as admin at /auth/login', (done) => {
+    const loginData = {
+      email: 'admin@gmail.com',
+      password: 'check',
+    };
+
+    chai
+      .request(server)
+      .post('/auth/login')
+      .send(loginData)
+      .then((res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body.email).to.equal(loginData.email);
+        expect(res.body.isSuperAdmin).to.equal(true);
+        expect(res.body.token).to.be.a('string');
+        token = res.body.token;
+        done();
+      })
+      .catch(done);
+  });
+
+  // Step 2 - delete broadcast from DB
+  it('delete broadcast from DB at /broadcast/:id', (done) => {
```

Comment:
would be good if you first add a broadcast then delete that

Reply:
ok